pkg/v1/tkg/managementcomponents/helper.go

```diff
@@ -62,6 +62,7 @@ func GetTKGPackageConfigValuesFileFromUserConfig(managementPackageVersion string
 				TanzuAddonsManager: TanzuAddonsManager{
 					FeatureGates: AddonsFeatureGates{
 						ClusterBootstrapController: true,
+						PackageInstallStatus:       true,
```

Comment:
Question: Why these are ```PackageInstallStatus``` instead of ```PackageInstallStatusController```?

Reply:
Thanks for this question--I have no idea. Let me look into it. I wonder if @shyaamsn knows real quick? But I'll still look into it.

Reply:
Its because this is what is specified in the addons-manager package values.yaml https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/addons-manager/bundle/config/values.yaml#L18

Seems like this should have been packageInstallStatusController. But dont think its a good idea to change now